explorer/client/src/pages/object-result/ObjectLoaded.tsx

```diff
@@ -83,6 +84,19 @@ function ObjectLoaded({ data }: { data: DataType }) {
     const isPublisherGenesis =
         data.objType === 'Move Package' && data?.publisherAddress === 'Genesis';
 
+    const structProperties = Object.entries(viewedData.data?.contents)
+        .filter(([_, value]) => typeof value == 'object')
+        .filter(([key, _]) => key !== 'id');
+
+    // Add struct properties to the properties list
+    let structPropertiesTitle;
+    let structPropertiesContent;
+    if (structProperties.length > 0) {
+        const structPropertiesData = Object.values(structProperties);
+        structPropertiesTitle = structPropertiesData[0][0];
```

Comment:
If there are multiple struct properties, we should display them all instead of just the first one